concrete/blocks/image/view.css

```diff
@@ -0,0 +1,9 @@
+img.ccm-image-center {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+img.ccm-image-right {
+    margin-left: auto;
+    margin-right: 0;
+}
```

Comment:
You are, of course, right.

I checked whether display:block was already being applied from some other class, and it was. However, as with my previous oversight, this was coming from bootstrap-related classes defined in the elemental theme. (Why a core block applies theme-specific classes that aren't required to be defined in all themes evades me, but that's another story.)

I prepended img to narrow the scope of the selectors (since I've had trouble with greedy selectors), although I know it shouldn't be necessary. There are precedents in the c5 code either way. I've now removed the img.